wiki/Main_Page/sv.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: main_page
+no_native_review: true
```

Comment:
Can the main page have this mark @cl8n ?

Reply:
sure, it's not displayed on the website anyway